smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/PackageManifestGenerator.kt

```diff
@@ -27,49 +30,64 @@ class PackageManifestGenerator(val ctx: ProtocolGenerator.GenerationContext) {
                 }
 
-                val externalDependencies = dependencies.filter { it.getProperty("url", String::class.java).get().isNotEmpty() }
-                val dependenciesByURL = externalDependencies.distinctBy { it.expectProperty("url", String::class.java) }
+                val externalDependencies = dependencies.filter {
+                    it.getProperty("url", String::class.java).getOrNull() != null ||
+                        it.getProperty("scope", String::class.java).getOrNull() != null
+                }
+                val dependenciesByURL = externalDependencies.distinctBy {
+                    it.getProperty("url", String::class.java).getOrNull()
+                        ?: "${it.getProperty("scope", String::class.java).get()}.${it.packageName}"
+                }
 
                 writer.openBlock("dependencies: [", "],") {
-                    dependenciesByURL.forEach { dependency ->
-                        writer.openBlock(".package(", "),") {
-
-                            val localPath = dependency.expectProperty("localPath", String::class.java)
-                            if (localPath.isNotEmpty()) {
-                                writer.write("path: \$S", localPath)
-                            } else {
-                                val dependencyURL = dependency.expectProperty("url", String::class.java)
-                                writer.write("url: \$S,", dependencyURL)
-                                writer.write("from: \$S", dependency.version)
-                            }
-                        }
-                    }
+                    dependenciesByURL.forEach { writePackageDependency(writer, it) }
                 }
 
                 val dependenciesByTarget = externalDependencies.distinctBy { it.expectProperty("target", String::class.java) + it.packageName }
 
                 writer.openBlock("targets: [", "]") {
                     writer.openBlock(".target(", "),") {
                         writer.write("name: \$S,", ctx.settings.moduleName)
-                        writer.openBlock("dependencies: [", "]") {
-                            for (dependency in dependenciesByTarget) {
-                                writer.openBlock(".product(", "),") {
-                                    val target = dependency.expectProperty("target", String::class.java)
-                                    writer.write("name: \$S,", target)
-                                    writer.write("package: \$S", dependency.packageName)
-                                }
-                            }
+                        writer.openBlock("dependencies: [", "],") {
+                            dependenciesByTarget.forEach { writeTargetDependency(writer, it) }
+                        }
+                        writer.openBlock("resources: [", "]") {
+                            writer.write(".process(\"Resources\")")
```

Comment:
Each service client has a `Package.version` file in the `Resources` subfolder that needs to be included with the service client.